### PR TITLE
Update rundown of AWS accounts

### PR DESCRIPTION
**Pull request by fureigh**
See commit messages for details.

One outstanding question, not created by this PR but not yet addressed in it: what/where is the "here" referenced in "any support tickets should be filed here"?

### CODE REVIEW

**Comment by jcscottiii** [comment]
Looks good! Let's wait until E/W is shutdown completely then we can merge this.

**Review by mogul — APPROVED** [review]

**Pull request merged**
